### DISCUSSION TITLE
How to tell the most recent version of Cloudlog

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1571] [Q&A] [answered]
Hi guys,

How I can tell if I can have the most recent version of Cloudlog?
I have a cron job setup, but how to be 100% sure that it works?
Is it possible to see some version number somewhere, something? Even from inside?

Thanks, 
Dragan

**Comment by AndreasK79** [comment, marked as answer]
Hi Dragan. Nah, not really, as the version numbers aren't updated very often.

If you are on the master branch, you can be pretty sure that you are on the latest version, as this receives very few updates.
Station_logbooks is the developement branch, which when released, will be version 2. This will then replace the master branch.

**Reply by 4O4A** [reply]
Thank you, Andreas!+
